HOWTOSTART.md

```diff
@@ -0,0 +1,334 @@
+# Setting up Development Environment for Origam
+The following is a guide to help you set up Origam development environment. You will need it if you wish to contribute to Origam backend or frontend. It will be quite helpful if you are going to develop your own plugin for Origam too.
+
+Origam can be developed on Windows only because the Architect application is written using Windows Forms. 
+If you wish to run the server and front end only you can probably do that on Linux and Mac too.
```

Comment:
This relates to the comment above. It sounds almost like we neither compiled our own product nor we understand it.

I would kindly ask guys developing Origam on Mac for an input here - A section about what you need to develp Origam on Mac would be nice.